bot/tests/test_reporter_phabricator.py

```diff
@@ -67,6 +67,12 @@
 If you see a problem in this automated review, [please report it here](https://bugzilla.mozilla.org/enter_bug.cgi?product=Firefox+Build+System&component=Source+Code+Analysis&short_desc=[Automated+review]+UPDATE&comment=**Phabricator+URL:**+https://phabricator.services.mozilla.com/...&format=__default__).
 
 You can view these defects on [the code-review frontend](https://code-review.moz.tools/#/diff/42) and on [Treeherder](https://treeherder.mozilla.org/#/jobs?repo=try&revision=deadbeef1234).
+
+In our previous code coverage analysis run, we found some files which had no coverage and are being modified in this patch:
```

Comment:
Could we move it before `If you see a problem in this automated review`

Reply:
Done